test/local/user_agent.js

```diff
@@ -559,6 +559,36 @@ describe('userAgent', () => {
   )
 
   it(
+    'recognises new Firefox-iOS user agents',
+    () => {
+      parserResult = {
+        ua: {
+          family: 'Other'
+        },
+        os: {
+          family: 'Other'
+        },
+        device: {
+          family: 'Other'
+        }
+      }
+      const context = {}
+      const userAgentString = 'Firefox-iOS-FxA/6.0b42 (iPhone 6S; iPhone OS 10.3) (Firefox)'
+      const result = userAgent.call(context, userAgentString, log)
+
+      assert.equal(result.uaBrowser, 'Firefox')
+      assert.equal(result.uaBrowserVersion, '6.0')
+      assert.equal(result.uaOS, 'iOS')
```

Comment:
One thing I noticed with this "uaOS" field - when we parse the standard front-end UA string is comes out as "iPod" or "iPad" or similar, but on the backend we always use "iOS".  Should we try to standardize on just "iOS" everywhere to simplify queries?  We currently do `uaOS LIKE 'iP%'` in a few redash queries to work around this IIRC.